src/services/jwt.service.js

```diff
@@ -0,0 +1,36 @@
+const jwt = require('jsonwebtoken');
+
+const sign = (user) => {
+  const token = jwt.sign(user, process.env.JWT_KEY, { expiresIn: '30m' });
+
+  return token;
+};
+
+const verify = (token) => {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (e) {
+    return null;
+  }
+};
+
+const signRefresh = (user) => {
+  const token = jwt.sign(user, process.env.JWT_REFRESH_KEY);
```

Comment:
Consider adding an expiration time for the refresh token to enhance security. This will ensure that refresh tokens are not valid indefinitely.